main.tf

```diff
@@ -1,13 +1,20 @@
-resource "random_integer" "example" {
-  count = module.this.enabled ? 1 : 0
-
-  min = 1
-  max = 50000
-  keepers = {
-    example = var.example
+locals {
+  service_control_policy_content = {
+    Version   = "2012-10-17"
+    Statement = var.service_control_policy_statements
   }
 }
 
-locals {
-  example = format("%v %v", var.example, join("", random_integer.example[*].result))
+resource "aws_organizations_policy" "this" {
+  count       = module.this.enabled ? 1 : 0
+  name        = module.this.id
+  description = var.service_control_policy_description
+  content     = jsonencode(local.service_control_policy_content)
```

Comment:
We should use an `aws_iam_policy_document` here so that we get validation.

Reply:
Unfortunatly this is not possible since we cannot `for-each` on the dynamic statements in a `data resource`

Reply:
updated to use `aws_iam_policy_document`